src/ert/job_queue/queue.py

```diff
@@ -289,7 +299,7 @@ async def _publish_changes(
             ]
         )
         while events:
-            await asyncio.wait_for(ee_connection.send(to_json(events[0])), 60)
```

Comment:
:+1: Looking at `WebSocketCommonProtocol.send()`, it actually discourages using `asyncio.wait_for.`